src/pygeoprocessing/geoprocessing.py

```diff
@@ -2722,10 +2751,23 @@ def transform_bounding_box(
     target_ref = osr.SpatialReference()
     target_ref.ImportFromWkt(target_ref_wkt)
 
-    transformer = osr.CoordinateTransformation(base_ref, target_ref)
+    # Create a coordinate transformation
+    # In GDAL 3.0 spatial references no longer ignore Geographic CRS 
+    # Axis Order and conform to Lat first, Lon Second. Transforms
+    # expect (lat, lon) order as opposed to the GIS friendly (lon, lat). 
+    # See https://trac.osgeo.org/gdal/wiki/rfc73_proj6_wkt2_srsbarn Axis order
+    # issues. SetAxisMappingStrategy(osr.OAMS_TRADITIONAL_GIS_ORDER) swaps 
+    # the axis order, which will use Lon,Lat order for Geographic CRS, 
+    # but otherwise leaves Projected CRS alone
```

Comment:
Same comment as above

Reply:
Done in [c9f0d6](https://github.com/natcap/pygeoprocessing/pull/31/commits/c9f0d66d3513cb8559367280e605b008d244f30c)